README.md

```diff
@@ -101,6 +103,7 @@ Please only use lower case schema names with this adapter.
 - [ldap](https://trino.io/docs/current/security/authentication-types.html) - Specify username in `user` and password in `password`
 - [kerberos](https://trino.io/docs/current/security/kerberos.html) - Specify username in `user`
 - [jwt](https://trino.io/docs/current/security/jwt.html) - Specify JWT token in `jwt_token`
+- [certificate](https://trino.io/docs/current/security/certificate.html) - Specify a client certificate in `client_certificate` and private key in `client_private_key`
```

Comment:
@hovaesco @Haunfelder  How does this change relate to https://github.com/starburstdata/dbt-trino/pull/24/files#r780898480 ?

Reply:
`cert` parameter is used to specify client certificate path to connect with SSL/TLS.

`certificate` based authentication is used to validate certs provided by the Trino and Trino validates certs provided by a client.

These two don't interfere each other.

Reply:
I like the proposal, haven't seen a similar structure for other adapters, but it's worth considering. 

The reason behind `cert` name was to align with `trino-python-client` naming convention.